tests/bug/test_gh_1928.py

```diff
@@ -0,0 +1,41 @@
+import json
```

Comment:
Please don't introduce a new layout pattern that goes against what the project already has in place. Also, I don't believe in the test only PRs, so would prefer to feature with test together.

Reply:
This should go and follow the existing ``{posargs}`` syntax from https://github.com/tox-dev/tox/blob/rewrite/tests/config/loader/ini/replace/test_replace_posargs.py